project/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@anev/ts-mountebank",
-  "version": "1.8.2",
+  "version": "1.8.3",
```

Comment:
No need to update the version - I will release once there is a package update in the library or a functional change in there.

Reply:
Roger. Let me undo that change. Sorry.